src/components/VContentReport/VContentReportForm.vue

```diff
@@ -60,12 +60,12 @@
 
         <div class="mb-4 min-h-[7rem]">
           <VDmcaNotice
-            v-if="selectedReason === reasons.DMCA"
+            v-if="media.foreign_landing_url && selectedReason === reasons.DMCA"
```

Comment:
Thank you for fixing this! The warnings were bothering me. I guess they are fired before the 'image' object is loaded, when the image detail is pre-fetched.